xmtp_id/src/scw_verifier.rs

```diff
@@ -39,7 +49,7 @@ impl ERC1271Verifier {
     /// * `contract_address` - Address of the ERC1271 wallet.
     /// * `block_number` - Block number to verify the signature at.
     /// * `hash`, `signature` - Inputs to ERC-1271, used for signer verification.
-    pub async fn is_valid_signature(
+    pub async fn erc1271_is_valid_signature(
```

Comment:
Once we have tests for `erc6492_is_valid_signature` this function will go away, right?

Reply:
I believe so, erc6492 is a superset of erc1271.

Reply:
erc6492 also covers general ECDSA signature recovery, so we might end up aggregating `add_ecdsa_signature` & `add_erc1271_signature` into one `add_erc6492_signature`.

Reply:
It's true. It would work. But then we have to go to the network to validate every signature, which is going to be a lot slower.

We do also have a bunch of additional fields for SCW signatures (like block height) that don't really make sense for EOA sigs.

Reply:
Makes sense, `add_ecdsa_signature` is way faster